libs/game/game.ts

```diff
@@ -58,6 +58,7 @@ namespace game {
     }
 
     export function popScene() {
+        _scene._clearScene();
```

Comment:
is _scene guaranteed to be defined?

Reply:
Not if popScene was just called, but even if it was guaranteed I should have a check there